azure-pipelines.vrt-baseline.yml

```diff
@@ -26,7 +26,7 @@ jobs:
           fluentVersion: v9
           vrTestPackageName: '@fluentui/vr-tests-react-components'
           vrTestPackagePath: 'apps/vr-tests-react-components'
-          shouldBuildstorybookaddon: true
+          shouldBuildstorybookaddon: false
```

Comment:
all these configurations can be removed from the template as they will be never enabled after this is merged cc @TristanWatanabe

Reply:
Created a separate issue to follow up: #27010